### DISCUSSION TITLE
Issue with detecting mime type of datanorm files (.001)

### DISCUSSION THREAD

**Discussion by asioso** [pimcore/pimcore#14106] [Q&A / Support]
Hi, I have issue with datanorm files e.g. [datanorm.zip](https://github.com/pimcore/pimcore/files/10457842/datanorm.zip) (file is inside datanorm directory, file extension .001) in latest pimcore version.

1. in https://demo.pimcore.fun/ doesn't work:
![image](https://user-images.githubusercontent.com/48808653/213475215-94988f45-e1e0-4ff9-8071-71ed81a85e17.png)
getting type unknown, and can't replace the asset with the same file:
![image](https://user-images.githubusercontent.com/48808653/213475398-09c47ef4-1aba-4500-8f04-d699d503825c.png)

2. in https://demo.pimcore.com/ works and has correct type and can be replaced
![image](https://user-images.githubusercontent.com/48808653/213475596-4a7b7e45-aad4-4b25-96ef-267afef5ede3.png)

https://demo.pimcore.fun/ and  https://demo.pimcore.com/ have same pimcore version:
![image](https://user-images.githubusercontent.com/48808653/213475776-98bef2bb-b12b-450d-a50e-c6e5228cc136.png)

What is the difference ? How I can get same behaviour as on https://demo.pimcore.com/ ?

**Comment by fashxp** [comment]
Hmm, I think one of the differences between demo-fun and demo-com is, that the symfony messenger is not processed at demo-fun. 
Can that make the difference?

**Reply by asioso** [reply]
@fashxp thanks for reply. Not sure, we have also symfony messenger running, but doesn't work. What is strange is that detect correct type with Upload new version, but with initial upload not.

**Reply by asioso** [reply]
@fashxp maybe something in nginx configuration file ?
